api/src/main/java/net/md_5/bungee/api/SkinConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package net.md_5.bungee.api;
+
+/**
+ * Represents a player's skin settings.
+ */
+public interface SkinConfiguration
```

Comment:
If you go to Options and then Skin Customization there are various settings you can turn on and off.
![image](https://user-images.githubusercontent.com/5008081/29018699-64905ed2-7b5c-11e7-8d6e-52f462d2bf2e.png)

I don't know what this could be used for exactly but I'm sure someone will come up with something clever. I can imagine some use cases in MMO. If you feel this is a useless implementation then I can remove it. I only added it because it was one of the missing Settings. I am only working on this pull request to get access to the players ChatMode (referring to my original post at the top).